src/test/java/hudson/plugins/junitattachments/AttachmentPublisherPipelineTest.java

```diff
@@ -129,4 +129,32 @@ private String fileContentsFromResources(String fileName) throws IOException {
 
     }
 
+    @Test
+    public void testBothWellKnownFilenamesAndPatternAreAttached() throws Exception {
```

Comment:
Ok, awesome. Thanks @topikachu 

One last thing, can you move this method below `annotationDoesNotFailForPipeline()` instead to keep it consistent.